clinica/pipelines/engine.py

```diff
@@ -769,14 +770,16 @@ def copy2_add_ses(src, dst):
 
             # The following code is run if cross sectional subjects have been found
             if len(cross_subj) > 0:
-                cprint(
-                    msg=(
-                        f"{len(cross_subj)} subjects in your BIDS folder did not respect the longitudinal organisation "
-                        "from BIDS specification. Clinica does not know how to handle cross sectional dataset, but it "
-                        "can convert it to a Clinica compliant form (using session ses-M00)"
-                    ),
-                    lvl="warning",
+                msg = (
+                    f"{len(cross_subj)} subjects in your BIDS folder did not respect the longitudinal organisation "
+                    "from BIDS specification.\nThe subjects concerned are:\n\t- "
+                )
+                msg += "\n\t- ".join(cross_subj)
```

Comment:
Possible implementation of @ghisvail 's suggestion:

```python
max_subjects_displayed = 50
bound = min(len(cross_subj), max_subjects_displayed)
msg = (
    f"{len(cross_subj)} subjects in your BIDS folder did not respect the longitudinal organisation "
    f"from BIDS specification.\nThe subjects concerned are (showing only the first {bound}):\n\t- "
)
msg += "\n\t- ".join(cross_subj[:bound])
```

Reply:
Yes that looks like a good thing to do !